postfix/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 
 ### Changes
 
+* [IMPROVEMENT] option to run the check using postqueue -p
```

Comment:
If `1.1.0` is unreleased, then I guess this should be all below? We have two `1.1.0` entries in the changelog.